src/table/components/head/styles.ts

```diff
@@ -72,7 +72,6 @@ export const StyledHeadCellContent = styled(Box, {
   fontFamily: "inherit",
   "&&:hover, &&:focus-within": {
     "& .sn-table-head-menu-button": { opacity: 1 },
-    "& #nebula-table-utils-head-menu-button": { opacity: 1 },
```

Comment:
No need for this at all, since HeadCellMenu component always has the menu button visible